lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -46,7 +46,7 @@ namespace spv {
 
 typedef unsigned int Id;
 
-#define SPV_VERSION 10000
+#define SPV_VERSION 0x10000
 #define SPV_REVISION 2
```

Comment:
I hadn't realise at that time that this was the `spirv.hpp` from the SPIR-V registry.
It would be nice to have a repository for SPIR-V headers, as it would make it easier to update them and find if they have been updated.